### PR TITLE
Altering policies for SNS to allow add/removal of config subscription

### PR DESCRIPTION
**Pull request by chrisgilbert**
A small alteration to SNS IAM policies to allow listing of config subscriptions. This is for the killingserverssofly script.

### CODE REVIEW

File: tyr/policies/ec2.py [hunk 1]
```diff
@@ -211,8 +243,10 @@
         "sns:Subscribe"
       ],
       "Resource": [
-        "arn:aws:sns:us-east-1:761584570493:p-outpost",
-        "arn:aws:sns:us-east-1:761584570493:alyx3"
+        "arn:aws:sns:us-east-1:761584570493:prod-outpost",
```
Comment (wliao008) [inline tyr/policies/ec2.py]: I don't think `prod-outpost` exist?

File: tyr/policies/ec2.py [hunk 2]
```diff
@@ -242,8 +276,10 @@
         "sns:Subscribe"
       ],
       "Resource": [
-        "arn:aws:sns:us-east-1:761584570493:s-outpost",
-        "arn:aws:sns:us-east-1:761584570493:alyx3"
+        "arn:aws:sns:us-east-1:761584570493:stage-outpost",
```
Comment (wliao008) [inline tyr/policies/ec2.py]: `stage-outpost` doesn't exist as well i think

Comment (Z3r0Sum) [inline tyr/policies/ec2.py]: Yeah, curious as to why you changed the topic ARN?  Looks like the following exist for outpost:

```
t-outpost arn:aws:sns:us-east-1:761584570493:t-outpost
p-outpost arn:aws:sns:us-east-1:761584570493:p-outpost
s-outpost arn:aws:sns:us-east-1:761584570493:s-outpost
```

File: tyr/policies/ec2.py (hunk 1 above)
Comment (Z3r0Sum) [inline tyr/policies/ec2.py]: Can you remove this line?  It doesn't seem like test-outpost exists and you already have line for `t-outpost`.

```
t-outpost arn:aws:sns:us-east-1:761584570493:t-outpost
```

File: tyr/policies/ec2.py (hunk 2 above)
Comment (chrisgilbert) [inline tyr/policies/ec2.py]: Thanks, I have removed these now.  To be honest I was just worried there might be some using an old name - I should have just checked really.

**Pull request merged**